modules/css/layout.scss

```diff
@@ -22,11 +22,16 @@ div#board {
     align-items: center;
     grid-row-end: span 2;
     grid-column-end: span 4;
-    height: 2*$cardH + 30px;
+    max-height: $cardH + 20px;
     min-height: $cardH + 20px;
```

Comment:
I would say there's no need to put both max-height and min-height with similar values. Also, I don't think #board should be that small, it's now minimum 800px in master, it would be tiny, isn't it?

Reply:
Fixed max-height, it was a typo from me.